book/src/drivers.md

```diff
@@ -64,6 +64,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Intel PMEM-CSI](https://github.com/intel/pmem-csi) | `pmem-csi.intel.com` | v1.0 | A Container Storage Interface (CSI) driver for [PMEM](https://pmem.io/) from Intel | Persistent and Ephemeral | Read/Write Single Pod | Yes | Raw Block
 [Intelliflash Block Storage](https://github.com/DDNStorage/intelliflash-csi-block-driver) | `intelliflash-csi-block-driver.intelliflash.com` | v1.0, v1.1, v1.2 | A Container Storage Interface (CSI) Driver for Intelliflash  Block Storage | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion, Cloning, Topology
 [Intelliflash File Storage](https://github.com/DDNStorage/intelliflash-csi-file-driver) | `intelliflash-csi-file-driver.intelliflash.com` | v1.0, v1.1, v1.2 | A Container Storage Interface (CSI) Driver for Intelliflash  File Storage | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion, Cloning, Topology
+[ionir ](https://github.com/ionir-cloud) | `ionir` | v1.2 | A Container Storage Interface (CSI) Driver for [ionir](https://www.ionir.com/) Kubernetes-Native Storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Cloning
```

Comment:
I checked "https://github.com/ionir-cloud" but it is not clear to me which is the document on how to use this CSI driver.  Can you point me to the user guide?

Reply:
Hi Xing
There are multiple examples on the repo including how to use with various applications and solutions

Is there something specific we should publish as part of it?

We have a user guide that is customer facing ... should we upload it to the repo as well?

Looking at other csi vendors it looked like this is enough

Either way happy to complete the gap and push this

Reply:
Can you share a link here so that I can take a look?

Reply:
that is weird, the repos are public you should see an "openshift" folder in there
also "https://github.com/ionir-cloud/examples" has few examples of applications and storage classes

Reply:
can you please check if the examples are visible for you @xing-yang ?

Reply:
Yes, I can see those examples.
Do you have document similar to the following that shows user how to deploy the CSI driver along with the sidecars?
https://github.com/kubernetes-csi/csi-driver-host-path/blob/master/docs/deploy-1.17-and-later.md#deployment

Reply:
Is this CSI driver deployed by the user or deployed automatically by a distro?

Reply:
It is installed by the user by script or operator today

Reply:
Okay